client/src/components/App/Content/LandingPage/InvestigationTable/TableFilter/TableFilter.tsx

```diff
@@ -23,7 +23,7 @@ const TableFilter = (props: Props) => {
         onFilterChange, onClose, 
         changeInactiveUserFilter, onSubStatusChange, inactiveUserFilter, 
         changeUnassginedUserFilter, unassignedUserFilter, 
-        timeRangeFilter, onTimeRangeFilterChange
+        timeRangeFilter, onTimeRangeFilterChange, updateDateFilter, nonContactFilter
```

Comment:
maybe break the row (put  updateDateFilter, nonContactFilter in new row)

Reply:
👍